### PR TITLE
Updated installation documentation

### PR DESCRIPTION
**Pull request by nfahlgren**
### Description

Updated installation instructions based on feedback from several users, including issues #213, #214, #215, and #216.
### Types of changes

This is an update to the documentation only:

* Simplified installation instructions
* Added primary Anaconda instructions
* Collapsed script-based method into one section
* Updated heading levels
* Removed RedHat, virtualenv, and manual Ubuntu and OS X methods

### Checklist:

- [x] Relevant tests (and test data) have been added or updated and passed.
- [x] The documentation was added or updated.
- [x] Relevant issues were updated or added.

### CODE REVIEW

**Comment by nfahlgren** [comment]
@ic and @lchavez037 this pull request updates the installation documentation. Since you have both contributed to the guide I would value your feedback on the updated page. You can preview it at https://github.com/danforthcenter/plantcv/blob/0f7735c30f1f5b51660de36cabcf3cae29cd9cde/docs/installation.md

**Comment by maliagehan** [comment]
yay new simplified installation instructions

File: docs/installation.md [hunk 1]
```diff
@@ -2,310 +2,156 @@
...
-    - python-dateutil 2.6
-    - scikit-image 0.12
-    - scipy 0.18
-    - setuptools 28.6
-- OpenCV 2.4
-- SQLite 3.7
-- Git 1.8
+- Python (tested with versions 2.7 and 3.6)
+    - argparse
+    - cv2 (we recommend 2.14+ or 3.3+)
```
Comment (ic) [inline docs/installation.md]: The `cv2` version was min 2.4 before, and now 2.14. Is that ok?

File: docs/installation.md [hunk 2]
```diff
@@ -2,310 +2,156 @@
...
+bash scripts/setup.sh
 ```
 
-If everything is working, the following should run without errors:
-
-`python -c 'import plantcv'`
+The script guides you through the installation steps. Successful completion ends with a usage report.
 
-Or for more extensive tests:
+The script has been tested on [Ubuntu](http://www.ubuntu.com/) x86_64-bit 16_04 server edition and OSX 10.11.
```
Comment (ic) [inline docs/installation.md]: An extra update: Tested on macOS 10.12 too.

Comment (nfahlgren) [reply docs/installation.md]: Thanks!

File: docs/installation.md (hunk 1 above)
Comment (nfahlgren) [reply docs/installation.md]: Good catch, I confused the OpenCV major and minor versions. I corrected it to 2.4.14.

**Pull request merged**